formats/transaction/publisher/details.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "introductory_paragraph": {
+      "$ref": "#/definitions/body_html_and_govspeak"
+    },
+    "will_continue_on": {
+      "description": "Name of the website the Start button will lead the user to.",
+      "type": "string"
+    },
+    "transaction_start_link": {
+      "description": "Link the Start button will lead the user to.",
+      "type": "string",
```

Comment:
This could use one of the existing link definitions. The options: https://github.com/alphagov/govuk-content-schemas/blob/master/formats/definitions.json#L67-L78